evm/evm-processor/src/interfaces/evm.ts

```diff
@@ -35,6 +35,16 @@ export interface EvmTransaction extends _EvmTx, _EvmTxReceipt {
 }
 
 
+export interface Authorization {
```

Comment:
@eldargab do we need `Evm` prefix as for other entities?

Reply:
Prefix might be helpful, but don't forget to re-export `Authorization` in a prefix-less form.

I'm afraid `Authorization` is too ambiguous, though. May be call it like `EIP7702Authorization`?